frontend/src/pages/CreatePost.jsx

```
@@ -0,0 +1,439 @@
+                    className="absolute z-10 w-full mt-1 bg-white border border-gray-300 rounded-lg shadow-lg max-h-48 overflow-y-auto"
+                  >
+                    {isLoadingSkills && allSkills.length === 0 ? (
+                      <div className="p-2 text-center text-gray-500">Loading skills...</div>
+                    ) : filteredSkills.length > 0 ? (
+                      <ul>
+                        {filteredSkills.map((skill) => (
+                          <li 
+                            key={skill.id || skill.name} 
+                            className="px-4 py-2 hover:bg-indigo-50 cursor-pointer transition-colors flex items-center gap-2"
+                            onClick={() => handleSelectSkill(skill.name)}
+                          >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)